<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_step</class>
 <widget class="QWidget" name="add_step">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hlayout_action">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_action">
       <property name="text">
        <string>Action:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_action"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_expected">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_expected_result">
       <property name="text">
        <string>Expected result:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_expected_result"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_requirements" stretch="0,0">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>250</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_requirement">
       <property name="text">
        <string>Affected requirements:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lw_requirements">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_comment">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_comments">
       <property name="text">
        <string>Comments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_comments"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_btn_step">
     <property name="leftMargin">
      <number>300</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_accept">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
